models/DataObject/Listing/Concrete.php

```diff
@@ -54,7 +54,7 @@ abstract class Concrete extends Model\DataObject\Listing
      */
     public function __construct()
     {
-        $this->initDao('\\Pimcore\\Model\\DataObject\\Listing\\Concrete');
+        $this->initDao(__CLASS__);
```

Comment:
not sure if right, but doesn't ```__CLASS__``` return the actual class also on classes that extend Listing\Concrete?

Reply:
No, __CLASS__ always points to the class where this statement is in.